release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -514,6 +514,19 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-nw-metallb"]
+==== Changes to MetalLB
+
+With this update, MetalLB uses `FRR-K8s` as the default backend. 
+Previously this was an optional feature available in Technology Preview.
```

Comment:
Add comma after "Previously".

Reply:
thanks for the review @lpettyjo 
I've corrected that.